client/src/pages/App/Parents/AddRecommendationModal/AddRecommendationModal.tsx

```diff
@@ -0,0 +1,90 @@
+import {
+  Button,
+  Form,
+  FormField,
+  Header,
+  Modal,
+  ModalContent,
+  ModalDescription,
+  ModalHeader,
+} from "semantic-ui-react";
+import { useMutation } from "react-query";
+import { FieldValues, useForm } from "react-hook-form";
+
+import { ModalAddReviewProps } from "./RecommendationModalInterfaces";
+import { addRecommendation } from "./recommendationServices";
+
+const AddRecommendationModal = ({
+  isOpen,
+  setIsOpen,
+  parentId,
+  babysitterId,
+  babysitterName,
+}: ModalAddReviewProps) => {
+  const {
+    register,
+    handleSubmit,
+    reset,
+    formState: { isSubmitting },
```

Comment:
on submit please show a popup message that says thanks for your review and close the modal